### ISSUE TITLE
How to adjust whitespace between columns in gridString

### ISSUE THREAD

**Issue by hasufell** [muesli4/table-layout#10]
```hs
  let
    formatted =
      gridString
          [ column expand left def def
          , column expand left def def
          , column expand left def def
          , column expand left def def
          , column expand left def def
          ]
        . ([color Green "", "Tool", "Version", "Tag", "Notes"]:)
        . fmap
            (\ListResult {..} ->
              [ if
                | lSet       -> (color Green "✔✔")
                | lInstalled -> (color Green "✓")
                | otherwise  -> (color Red "✗")
              , fmap toLower . show $ lTool
              , T.unpack . prettyVer $ lVer
              , intercalate "," $ ((fmap . fmap) toLower . fmap show $ lTag)
              , if fromSrc then (color Blue "compiled") else mempty
              ]
            )
         $ lr
  putStrLn $ formatted
```

Trying to figure out how to make the columns a little farther from each other. Neither `fixedUntil`, nor `expandUntil` is what I'm looking for. What I'm looking for is: **setting the minimum whitespace between two columns**.

**Comment by muesli4** [comment]
`fixedUntil` and `expandUntil` are just for the cell-internal layout. Basically, `gridString` will use a default of one space and one newline as horizontal and vertical separators respectively. _The documentation should probably be improved here_.

What you can do instead is using the `grid` function in combination with `unlines` and `fmap $ concat . intercalate (replicate n ' ')`. _These simple functions should probably be in the library_.

If there is a high demand for more flexible spacing then the respective functions for grids (and perhaps tables) should also be provided.

Also in regard to colors: With the most recent version from hackage, color formatting characters are detected as printable character. The result is wrong length measurements. The bug has been fixed recently. See [here](https://github.com/muesli4/table-layout/issues/4#issuecomment-569075594) on how to use it. I will probably release a new version soon™. This will introduce quite a few changes to the API and I'm still not clear about all of the details and lack the time at the moment.

**Comment by muesli4** [comment]
Fixed as of f243821a3e5c5935903bb03fee1992c6109be3c2.
